src/components/institution/InstitutionAutocomplete.tsx

```diff
@@ -8,6 +8,7 @@ import { InstitutionUnitBase } from '../../types/institution.types';
 
 interface InstitutionAutocompleteProps extends Pick<TextFieldProps, 'disabled' | 'error' | 'helperText'> {
   institutions: InstitutionUnitBase[];
+  label?: string;
```

Comment:
```suggestion
interface InstitutionAutocompleteProps extends Pick<TextFieldProps, 'disabled' | 'error' | 'helperText' | 'label'> {
  institutions: InstitutionUnitBase[];
```

Reply:
bruk `label` fra `TextFieldProps` i stedet for å lage egen

Reply:
smart!